code/modules/hydroponics/gene_modder.dm

```diff
@@ -5,7 +5,8 @@
 	icon_state = "dnamod"
 	density = TRUE
 	circuit = /obj/item/circuitboard/machine/plantgenes
-	pass_flags = PASSTABLE
+	pass_flags = PASSTABLE | LETPASSTHROW
+	flags_1 = DEFAULT_RICOCHET_1 | ON_BORDER_1 // apparently this works???
```

Comment:
why is this on border it shouldn't be on border it's not a border object

Reply:
buddy i got no fuckin idea but throw impact code fucking SUCKS